test_project/project/settings.py

```diff
@@ -55,7 +55,10 @@
 
 # Database
 
-DATABASES = {"default": dj_database_url.config(default="sqlite://db.sqlite3")}
+DATABASES = {
+    "default": dj_database_url.config(default="sqlite://:memory:"),
+    "test": dj_database_url.config(default="sqlite://:memory:"),
+}
```

Comment:
I don't think we want to use in-memory instances here — the `test_project` is meant to be played with locally, with persisting data.

So I think we want this instead…?

```suggestion
DATABASES = {
    "default": dj_database_url.config(default="sqlite:///db.sqlite3"),
    "test": dj_database_url.config("TEST_DATABASE_URL", default="sqlite://test.sqlite3"),
}
```